Demo/Fullscreen/FavoriteAdsListView/FavoriteAdsDemoDataSource.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+struct AdsSection {
```

Comment:
Should we call it `FavoriteAdsSection`, just to avoid possible conflicts in future.

Reply:
Yeah, I agree